example/example_gradient_based_opt.jl

```diff
@@ -81,9 +124,15 @@ function exampleOPT()
         return size_ac(x...)
     end
 
+    function eval_g(x...)
+        return size_constraint(x...)
+    end
+
     register(model, :eval_f, size(initial)[1], eval_f, fdiff_all!; autodiff = false)
+    register(model, :eval_g, size(initial)[1], eval_g, fdiff_all_for_constraint!; autodiff = false)
 
     @NLobjective(model, Min, eval_f(x...))
+    @NLconstraint(model, eval_g(x...) >=0)
```

Comment:
Shouldn't the constraint be that `eval_g`$\leq 0$? For example, with `(ac.parg[igWfuel]/ac.parg[igWfmax] - 1.0)` we want `ac.parg[igWfuel]<ac.parg[igWfmax]` so that  `(ac.parg[igWfuel]/ac.parg[igWfmax] - 1.0)`<0